node/src/chain_spec.rs

```diff
@@ -231,7 +204,7 @@ pub fn harbor_testnet_config(id: ParaId) -> ChainSpec {
 }
 
 pub fn rococo_parachain_config(id: ParaId) -> ChainSpec {
-    let mut properties = Map::new();
+    let mut properties = sc_chain_spec::Properties::new();
     properties.insert("tokenSymbol".into(), "DHX".into());
     properties.insert("tokenDecimals".into(), 18.into());
     let boot_nodes = vec![];
```

Comment:
what has a parachain id of `2` (i.e. see our code `para_id: 2_u32.into(),`)

Reply:
are we using this function for both Rococo and ChaChaCha? (see our code `relay_chain: "rococo-chachacha".into(),`)

Reply:
do we want to use `DHX` as the token symbol on our Rococo and ChaChaCha parachain testnets?

Reply:
suggest changing 
```
    ChainSpec::from_genesis(
        "DataHighway",
        "datahighway",
```
to our parachain specific name like
```

    ChainSpec::from_genesis(
        "DataHighway Spreehafen Parachain Testnet",
        "datahighway_spreehafen",
```

since our testnet parachain network name for Rococo and ChaChaCha is Spreehafen, and our mainnet parachain network name for Kusama will be Tanganika

we will be using token symbol DHX on Tanganika
please check what token symbol we should use on Spreehafen

Reply:
> what has a parachain id of `2` (i.e. see our code `para_id: 2_u32.into(),`)? is this common to both Rococo and ChaChaCha? elsewhere in this PR we've used `2000`

shouldn't this be `para_id: id.into(),`, like in the substrate-parachain-template code